src/core/controllers/orbit-state.js

```diff
@@ -50,9 +36,9 @@ export default class OrbitState {
     width, // Width of viewport
     height, // Height of viewport
     distance, // From eye to target
-    pitchAngle, // Rotation around x axis
-    orbitAngle, // Rotation around orbit axis
-
+    rotationX, // Rotation around x axis
```

Comment:
personally, I agree with the original naming that _pitch_ is relatively easier to understand than _X_.
https://social.sbrick.com/custom/wiki/designer/latest/camera_pitch_yaw_roll.png

but as long as we have documented this clearly it should be fine.

Reply:
On this same topic, we should also document that, OrbitAxis can't be x-axis.